app/src/types/connectorsConfig.ts

```diff
@@ -0,0 +1,33 @@
+export type InputType = {
```

Comment:
is that necessary ? because we are going to maintain those type each and every time we have a new connector. Does the config is not about avoiding this ?